### PR TITLE
Add CI providers to community resources page.

### PR DESCRIPTION
**Pull request by russellb**

### CODE REVIEW

**Comment by russellb** [comment]
@maelk https://deploy-preview-90--metal3.netlify.com/community-resources.html

let me know if you'd like to add anything else!

**Comment by iranzo** [comment]
@russellb should we add Netlify there then? So far they were listed in privacy page but if we're listing other 'thank you' probably makes more sense here.

**Comment by russellb** [comment]
> @russellb should we add Netlify there then? So far they were listed in privacy page but if we're listing other 'thank you' probably makes more sense here.

good point.  and travis-ci while we're at it ...

**Comment by iranzo** [comment]
If you handle via this PR, remove the content from privacy, update the 'ammendment date' on that one, and ad the paragraph on this one :)

File: community-resources.html [hunk 1]
```diff
@@ -87,5 +87,12 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">Get Connected</h2>
 
                     Blog</a></li>
              </ul>
+           <header class="mk-main__header">
+           <h2 class="mk-heading mk-heading--xl mk-m-border">Sponsors of Testing Resources</h2>
+           </header>
+             <ul>
+                 <li><a href="http://www.nordix.org">Nordix</a>. Details <a href="https://github.com/metal3-io/metal3-docs/blob/master/ci/nordix-jenkins.md">here</a>.</li>
```
Comment (maelk) [inline community-resources.html]: Should it be https ?

Comment (russellb) [reply community-resources.html]: done, thanks

**Comment by russellb** [comment]
Latest available here: https://deploy-preview-90--metal3.netlify.com/community-resources.html

**Pull request merged**